app/views/StatusView/index.tsx

```diff
@@ -149,6 +146,19 @@ const StatusView = (): React.ReactElement => {
 
 	const statusType = Accounts_AllowInvisibleStatusOption ? STATUS : STATUS.filter(s => s.id !== 'offline');
 
+	const FooterComponent = () => (
+		<View style={styles.footerComponent}>
+			<Button
+				testID='status-view-submit'
+				disabled={
+					status === user.status && ((!!user.statusText && user.statusText === statusText) || (!user.statusText && !statusText))
```

Comment:
Are you sure this is working?
It's hard to read for my tired eyes 👀

Reply:
updated the code to be easy to read.